src/jdk.incubator.code/share/classes/jdk/incubator/code/internal/ReflectMethods.java

```diff
@@ -135,14 +146,16 @@ public static ReflectMethods instance(Context context) {
     private final Lower lower;
     private final TypeEnvs typeEnvs;
     private final Flow flow;
+    private final JavaFileManager fileManager;
     private final CodeReflectionSymbols crSyms;
     private final boolean dumpIR;
     private final boolean lineDebugInfo;
     private final CodeModelStorageOption codeModelStorageOption;
 
     private TreeMaker make;
     private ListBuffer<JCTree> classOps;
-    private Symbol.ClassSymbol currentClassSym;
+    private SequencedMap<String, Op> ops;
+    private Symbol.ClassSymbol currentClassSym, synthClassSym;
```

Comment:
Suggest finding a better name and declaring on a separate line.
e.g., rename to `codeModelsClassSym`.